docker-compose.yml

```diff
@@ -1,15 +1,51 @@
 version: '3.8'
 
```

Comment:
The `version` attribute is obsolete and is ignored by `docker compose`. Please remove it to avoid an confusion.